### ISSUE TITLE
Cannot change url

### ISSUE THREAD

**Issue by Resterday** [sjvc/Home-Assistant-Launcher#5]
I removed the url from the app so that it would be full screen. I read the warning and was ok with not being able to make changes without uninstalling the app, but even after clearing cache, removing data, deleting the original donloaded .apk, and uninstalling the app; when I re-download the .apk file and install, it always goes back to my old url / lock screen.

**Comment by sjvc** [comment]
Data may be restored after reinstalling apk because Google automatically backups data in your account. Just go remove data (without uninstall it), and re-open. URL bar should be visible again. Is it?

**Comment by Resterday** [comment]
Thanks for the quick reply. After reading your comment, I tried again and at first I was still stuck, but I eventually got it by doing the following(I have Google pixel):

1. Open app settings and navigate to "assistant"
2. Select "force stop" (if I didn't do this step it didn't matter how many times I cleared data or uninstalled the app, old data would repopulate)
3. Select "clear data"
4. Opened the app back up

.
